src/scroll-timeline-css.js

```diff
@@ -53,14 +53,23 @@ function initMutationObserver() {
       // Most likely we won't be able to fetch resources from other origins.
       return;
     }
-    fetch(linkElement.getAttribute('href')).then(async (response) => {
+
+    // Stop loading the stylesheet to ensure that we finished parsing it before it's
+    // loaded by the browser (and that animations are only started after
```

Comment:
at this point it's mostly already parsed by the browser and interpreted, only the layouting isn't fully done yet.